### PR TITLE
Update publications from MOX

### PR DESCRIPTION
**Pull request by elauksap**
Should solve #379.

### CODE REVIEW

**Review by bangerth — APPROVED** [review]

File: publications-2022.bib [hunk 1]
```diff
@@ -1,5 +1,89 @@
...
+    journal = {Discrete and Continuous Dynamical Systems - S},
+    volume = {15},
+    number = {8},
+    pages = {2391-2427},
+    year = {2022},
+    doi = {10.3934/dcdss.2022052}
+}
+
+@article{regazzoni2022emcirculation,
+    author = {F. Regazzoni and M. Salvador and P.C. Africa and M. Fedele and L. Dedè and A. Quarteroni},
```
Comment (bangerth) [inline publications-2022.bib:41]: The system complains about the non-ASCII character in Dede's name. Can you fix this?

Comment (elauksap) [reply publications-2022.bib]: Done! Thanks.

File: publications-2022.bib [hunk 2]
```diff
@@ -1,5 +1,89 @@
...
+    author={Piersanti, Roberto and Regazzoni, Francesco and Salvador, Matteo and Corno, Antonio F and Vergara, Christian and Quarteroni, Alfio and others},
+    journal={Computer Methods in Applied Mechanics and Engineering},
+    volume={391},
+    pages={114607},
+    year={2022},
+    publisher={Elsevier}
+}
+
+@article{vergara2022,
+    author = {Vergara, Christian and Stella, Simone and Maines, Massimiliano and Africa, Pasquale Claudio and Catanzariti, Domenico and Demattè, Cristina and Centonze, Maurizio and Nobile, Fabio and Quarteroni, Alfio and Del Greco, Maurizio},
```
Comment (marcfehling) [inline publications-2022.bib:22]: There are more :)

File: publications-2022.bib [hunk 3]
```diff
@@ -1,5 +1,89 @@
...
+}
+
+@article{regazzoni2022emcirculation,
+    author = {F. Regazzoni and M. Salvador and P.C. Africa and M. Fedele and L. Dede' and A. Quarteroni},
+    title = {A cardiac electromechanical model coupled with a lumped-parameter model for closed-loop blood circulation},
+    journal = {Journal of Computational Physics},
+    volume = {457},
+    pages = {111083},
+    year = {2022}
+}
```
Comment (marcfehling) [inline publications-2022.bib:47]: We have two arxiv preprints in the 2020 publication list with the same title. Did both preprints become this article in the end?

Comment (elauksap) [reply publications-2022.bib:47]: Thanks for spotting! I removed the two older preprints.

**Review by marcfehling — APPROVED** [review]
The CI now fails because of an entry that you haven't touched. I assume it's because you've added the 2022 publications to the offline publication list. This is something we can address in a separate PR.

**Comment by elauksap** [comment]
Thanks @marcfehling!

It's only due to a missing comma, I suspect.

I can fix it here, or open a new PR once this gets merged if you prefer.

BTW, the following patch should solve the issue with that entry:
```
diff --git a/publications-2022.bib b/publications-2022.bib
index 4e0128b..0d3bc3c 100644
--- a/publications-2022.bib
+++ b/publications-2022.bib
@@ -409,8 +409,8 @@ author = {Richard Schussnig and Douglas R.Q. Pacheco and Thomas-Peter Fries},
   number={6},
   pages={1259-1293},
   year={2022},
-  publisher={Springer}
-  doi={10.1007/s00466-022-02147-0},
+  publisher={Springer},
+  doi={10.1007/s00466-022-02147-0}
 }
```

**Comment by marcfehling** [comment]
If you want to, you can also fix it yourself in this PR, sure :)

**Comment by elauksap** [comment]
Done! :)

**Review by marcfehling — APPROVED** [review]

**Pull request merged**